include/mbgl/ios/MGLMapView.h

```diff
@@ -404,6 +404,30 @@ IB_DESIGNABLE
 - (void)setZoomLevel:(double)zoomLevel animated:(BOOL)animated;
 
 /**
+ * The minimum zoom level the can be shown at.
+ */
+@property (nonatomic) double minimumZoomLevel;
+
+/**
+ * Changes the minimum zoom level of the map.
+ *
+ * @param minimumZoomLevel The new minimum zoom level.
+ */
+- (void)setMinimumZoomLevel:(double)minimumZoomLevel;
+
+/**
+ * The maximum zoom level the can be shown at.
+ */
+@property (nonatomic) double maximumZoomLevel;
```

Comment:
We should document that the default value of this property is 20 (but could be further constrained by the style?).